mne/time_frequency/multitaper.py

```diff
@@ -556,3 +560,74 @@ def _psd_multitaper(x, sfreq, fmin=0, fmax=np.inf, bandwidth=None,
     if ndim_in == 1:
         psd = psd[0]
     return psd, freqs
+
+
+@verbose
+def tfr_array_multitaper(epoch_data, frequencies, sfreq=1.0, n_cycles=7.0,
```

Comment:
can you uniformize the API. For example here sfreq is third param and not second as for the others